process/smartContract/process.go

```diff
@@ -172,7 +171,7 @@ func (sc *scProcessor) ExecuteSmartContractTransaction(
 		return err
 	}
 
-	sc.txFeeHandler.AddProcessedUTx(consumedFee)
+	sc.txFeeHandler.ProcessTransactionFee(consumedFee)
```

Comment:
we could use directly the intermediatetransactionhandler interface - adding the new txs, in the creating the block. No more need for UnsignedTxHandler interface.

Reply:
This is changed in next pull request.